content/en/blog/_posts/2024-07-xx-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,113 @@
+    scheduler-simulator/score-result: >-
+      {"node-jjfg5":{"ImageLocality":"0","NodeAffinity":"0","NodeResourcesBalancedAllocation":"52","NodeResourcesFit":"47","TaintToleration":"0","VolumeBinding":"0"},"node-mtb5x":{"ImageLocality":"0","NodeAffinity":"0","NodeResourcesBalancedAllocation":"76","NodeResourcesFit":"73","TaintToleration":"0","VolumeBinding":"0"}}
+    scheduler-simulator/selected-node: node-mtb5x
+```
+
+Users can also integrate their custom plugins or extenders into the debuggable scheduler and visualize their results. 
+Also, this debuggable scheduler can run standalone, e.g., on any Kubernetes cluster or in integration tests. 
+Those would be useful to custom plugin developers to test their plugins or examine their custom scheduler in a real cluster with better debuggability.
+
+## What are the use cases?
+
+1. **Cluser users**: Examine if your scheduling constraints (e.g., PodAffinity, PodTopologySpread) work as intended.
+2. **Cluster admins**: Assess how your cluster would behave with changes to the scheduler configuration.
+3. **Scheduler plugin developers**: Test your custom scheduler plugins or extenders, or use the debuggable scheduler in integration tests or development clusters.
+
+## Getting started
+
+The simulator only requires Docker to be installed on your machine; a Kubernetes cluster is not necessary.
+
+```
+git clone git@github.com:kubernetes-sigs/kube-scheduler-simulator.git
+cd kube-scheduler-simulator
+make docker_up
+```
+
+You can then access the simulator's web UI at http://localhost:3000.
+
+Visit [Kube-scheduler-simulator repository](sigs.k8s.io/kube-scheduler-simulator) for more details!
+
+## Getting involved 
+
+The Kube-scheduler-simulator is developed by Kubernetes SIG Scheduling. Your feedback and contributions are welcome!
+
+Open issues or PRs at [Kube-scheduler-simulator repository](sigs.k8s.io/kube-scheduler-simulator)
+Join the conversation on the #sig-scheduling slack channel.
+
+## Acknowledgments
+
+The simulator has been maintained by dedicated volunteer engineers, overcoming many challenges to reach its current form. 
+
+A big shout out to all [the awesome contributors](https://github.com/kubernetes-sigs/kube-scheduler-simulator/graphs/contributors)!
+
+![contributors](./contributors.png)
```

Comment:
Same for this image; consider using '[figure](https://gohugo.io/content-management/shortcodes/#figure)' shortcode. Refer to how this shortcode was used in a [previous blog post (here)](https://github.com/kubernetes/website/blob/fa563d70564885ae829abf6a228ed4d082a77f75/content/en/blog/_posts/2021-09-29-data-duplication-in-data-heavy-k8s-env.md?plain=1#L108).